src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,28 @@
 package mate.academy.hibernate.relations.util;
 
+import jakarta.persistence.Entity;
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
+@Entity
```

Comment:
Is there HibernateUtil table exist in Database?

Reply:
No(